_content/libraries.md

```diff
@@ -1,26 +1,49 @@
-## SPSP
-
-Build Interledger payments into your apps with the [ILP SPSP Client][1]. This library can connect to any SPSP receiver for account discovery and payment detail negotation. For more information on how to use SPSP, read ["SPSP: Simple Payment Setup Protocol"][2] on the Interledger Blog. 
-
-## Koa ILP
-
-If you use the [Koa framework][3] in Node.js, this library will allow you to monetize your API in only a few lines of code. For more information, [check out our Github][4] and [our blog post about it.][5]
-
-## ILP Fetch
-
-You can use [ILP Fetch][6] as a client to HTTP-ILP. ILP Fetch follows the [Fetch API][7], but allows the client to send payments. You can use it to interact with paid APIs on Interledger.
-
-## Moneyd XRP
-
-[Moneyd-XRP][8] allows you to connect to the Interledger network via XRP. It can be used on the livenet or the testnet with minimal configuration. We also have "Moneyd-ETH" and "Moneyd-Lightning" in the works.
-
-## More tools coming soon!
-
-[1]: https://github.com/interledgerjs/ilp-protocol-spsp
-[2]: https://medium.com/interledger-blog/spsp-simple-payment-setup-protocol-2028292e6925
-[3]: http://koajs.com/
-[4]: https://github.com/interledgerjs/koa-ilp
-[5]: https://medium.com/interledger-blog/http-ilp-paid-api-calls-with-interledger-fda53643a2eb
-[6]: https://github.com/interledgerjs/ilp-fetch
-[7]: https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
-[8]: https://github.com/interledgerjs/moneyd-xrp
+## Connectors
+
+### Java Connector
+An Interledger [Connector implemented in Java][1]. This project is actively maintained by [Xpring](https://xpring.io) 
+and recommended for new production deployments. 
+
+### Rust Connector
+An Interledger [Connector implemented in Rust][2]. This project is not actively maintained, but has a robust feature-set
+and is easy to use.
+
+### Javascript Connector
+An Interledger [Connector implemented in Javascript][3]. This project is not actively maintained, but has been 
```

Comment:
I imagine Matt might also want you to update this, but otherwise LGTM

Reply:
@kincaidoneil thanks for picking that up. Yeah please @sappenin , same as the other PR

Reply:
Doh! Sorry I missed that - fixed!